src/data/matrix/basic.lean

```diff
@@ -547,3 +577,13 @@ lemma row_mul_vec [semiring α] (M : matrix m n α) (v : n → α) :
 end row_col
 
 end matrix
+
+namespace ring_hom
+variables {α β : Type*} [semiring α] [semiring β]
+variables {m n o : Type u} [fintype m] [fintype n] [fintype o]
+
+lemma map_matrix_mul (M : matrix m n α) (N : matrix n o α) (i : m) (j : o) (f : α →+* β) :
```

Comment:
```suggestion
@[simp]
lemma map_matrix_mul (M : matrix m n α) (N : matrix n o α) (i : m) (j : o) (f : α →+* β) :
```
Maybe this is opportunistic...

Reply:
Again, doesn't work as a `simp` lemma because the head of the LHS is a metavariable.